Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/05-PersistingSyncUps/PersistingSyncUps.tutorial

```diff
@@ -46,22 +46,44 @@
         @Code(name: "SyncUpsList.swift", file: PersistingSyncUps-01-code-0003.swift)
       }
 
-      With that change the project should be compiling. It is worth noting that `fileStorage` only
+      If you try to compile the project now, you will see that it still fails where we try to 
+      mutate the `syncUps` shared property with the error `setter for 'syncUps' is unavailable: 
+      Use '$shared.withLock' to modify a shared value with exclusive access.`
+
+      @Step {
+        Shared values are secretly references (that is how data is shared), and it is perfectly 
+        possible to mutate a `Shared` property from inside an effect. 
+        So to make it thread-safe to mutate, we must use the `withLock` method on the projected 
+        value of the `syncUps` shared property when the value is mutated.
+        
+        @Code(name: "SyncUpsList.swift", file: PersistingSyncUps-01-code-0004.swift)
+      }
```

Comment:
📓 Here I explain how and why we use `withLock` now, trying to match your tone and vocabulary using excerpts from your documentation.